### PR TITLE
add radar tablet search

### PR DESCRIPTION
**Pull request by MaxProtos**

### CODE REVIEW

File: src-fe/src/app/libs/common-components/radar-search/radar-search.component.html [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+            <app-icon class="radar-search__reset-button text-input-postfix"
+                      (click)="resetSearchQuery()"
+                      size="xs"
+                      icon="close">
+            </app-icon>
+        </app-input-text>
+        <app-icon class="radar-search__search-icon radar-search__search-icon--tablet"
+                  [class.radar-search__search-icon--tablet-dark]="isDarkThemeActive"
+                  *ngIf="isTabletLandscapeView"
+                  (click)="mobileCollapseViewChange()"
```
Comment (clain-atomi) [inline src-fe/src/app/libs/common-components/radar-search/radar-search.component.html:31]: expandInput

File: src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts [hunk 2]
```diff
@@ -27,15 +30,41 @@ export class RadarSearchComponent {
...
 		return this.theme === ComponentTheme.Dark;
 	}
 
+	constructor(private viewModeService: ViewModeService) {}
+
+	public ngOnInit(): void {
+		this.destroy$ = new Subject<void>();
+		this.viewModeService.isTabletLandscapeView$.pipe(takeUntil(this.destroy$)).subscribe((isTabletLandscapeView: boolean) => {
+			this.isTabletLandscapeView = isTabletLandscapeView;
+			this.isMobileCollapsed = isTabletLandscapeView;
```
Comment (clain-atomi) [inline src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts:48]: is mobile...whaat?

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts (hunk 2 above)
Comment (MaxProtos) [reply src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts:48]: isCollapsed

File: src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts [hunk 3]
```diff
@@ -27,15 +31,50 @@ export class RadarSearchComponent {
...
+	}
+
+	public expandInput(): void {
+		if (this.isTabletLandscapeView && !Boolean(this.searchQuery)) {
+			this.isExpanded = !this.isExpanded;
+			if (this.isExpanded) {
+				this.textInput.enableFocusState();
+				setTimeout(() => {
+					this.textInput.enableFocusState();
+				}, 0);
```
Comment (clain-atomi) [inline src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts:68]: no need to set , 0, just setTimeout(() => ...);

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts (hunk 3 above)
Comment (clain-atomi) [reply src-fe/src/app/libs/common-components/radar-search/radar-search.component.ts:68]: we need to do serious dev testing for this feature

**Review by clain-atomi — APPROVED** [review]

**Pull request merged**